src/selfplay/tournament.cc

```diff
@@ -32,7 +32,6 @@
 #include "mcts/stoppers/factory.h"
 #include "neural/factory.h"
 #include "selfplay/game.h"
-#include "utils/optionsparser.h"
```

Comment:
Not sure why this was removed.

Reply:
I removed it by accident (though I have added it before) and it was still compiled well. I have checked, that including file is in the file tournament.h already - thus the removing is actually a kind of redundancy

Reply:
It can be good style to include headers you use rather than relying on transitive includes, although in this case since its already in the 'primary' header its less of an issue.
However my main recommendation for reverting is that the PR is cleaner if it doesn't contain extraneous file changes.

Reply:
Added it back.